notebooks/image_models/solutions/serving.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<ul><li>I find the word "concrete" here confusing. What about just removing the sentence "Still, it is not concrete enough."?</li><li>we can identify an actual function that we call --&gt; we can identify a specific model variant to be called</li><li>By default, <code>serving_default</code> is the name of the serving function --&gt; The tag of the default serving function is "serving_default"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thank you! I updated the first and second items. </p><p>In terms of third one, I replaced <code>tag</code> to <code>name</code> to avoid confusion between the SavedModel tag.</p>